app/models/setting/cfgmgmt.rb

```diff
@@ -0,0 +1,31 @@
+  def self.update_subnets_from_facts_types
+    {
+      'all' => _('All'),
+      'provisioning' => _('Provisioning only'),
+      'none' => _('None'),
+    }
+  end
+
+  def self.default_settings
+    [
+      set('matchers_inheritance', N_("Foreman matchers will be inherited by children when evaluating Lookups for Hostgroups, Organizations and Locations"), true, N_('Matchers inheritance')),
+      set('Default_parameters_Lookup_Path', N_("Foreman will evaluate host lookups in this order by default"), ["fqdn", "hostgroup", "os", "domain"], N_('Default parameters lookup path')),
+      set('interpolate_erb_in_parameters', N_("Foreman will parse ERB in parameters and lookup values"), true, N_('Interpolate ERB in parameters')),
+      set('create_new_host_when_report_is_uploaded', N_("Foreman will create the host when a report is received"), true, N_('Create new host when report is uploaded')),
+      set('create_new_host_when_facts_are_uploaded', N_("Foreman will create the host when new facts are received"), true, N_('Create new host when facts are uploaded')),
+      set('default_location', N_("Hosts created from uploaded facts that did not send a location fact will be placed in this location"), '', N_('Default location'), nil, { :collection => proc { Hash[Location.all.map { |loc| [loc[:title], loc[:title]] }] } }),
+      set('default_organization', N_("Hosts created from uploaded facts that did not send a organization fact will be placed in this organization"), '', N_('Default organization'), nil, {:collection => proc { Hash[Organization.all.map { |org| [org[:title], org[:title]] }] } }),
+      set('location_fact', N_("Hosts created from uploaded facts will be placed in the location this fact dictates. The content of this fact should be the full label of the location."), 'foreman_location', N_('Location fact')),
+      set('organization_fact', N_("Hosts created from uploaded facts will be placed in the organization this fact dictates. The content of this fact should be the full label of the organization."), 'foreman_organization', N_('Organization fact')),
+      set('update_subnets_from_facts', N_("Foreman will update a Host's subnet from its facts"), 'none', N_('Update subnets from facts'), nil, { :collection => proc { update_subnets_from_facts_types } }),
+      set('update_hostgroup_from_facts', N_("Foreman will update a Host's Hostgroup from its facts"), true, N_('Update hostgroup from facts')),
+      set('ignored_interface_identifiers', N_("Ignore interfaces that match these values during facts importing, you can use * wildcard to match names with indexes e.g. macvtap*"), IGNORED_INTERFACES, N_('Ignore interfaces with matching identifier')),
+      set('always_show_configuration_status', N_("All hosts will show a configuration status even when there is no configuration report for the Host"), false, N_('Always show configuration status')),
+    ]
+  end
+
+  def self.humanized_category
+    N_('Configuration management')
```

Comment:
Ideas for a better name? :thinking:

Reply:
Config management - I prefer this, tried locally and seemed OK
Config mgmt - not great not terrible
Cfgmgmt - hopefully not

Reply:
Let's start with a real human readable version (as you have it now). If we run into troubles, we can always revisit it.

Reply:
As per disscussion below what about What about `Facts and Parameters`?

Reply:
No, I think that's bad. If I wanted to look for Puppet options, I wouldn't look at `Facts and Parameters`.

Reply:
What about now? We have two categories `Facts` and `Configuration management` and both support "subcategories" (I'd extract that to a module, but just if I've understood correctly what we want)